config/locales/devise.nl.yml

```diff
@@ -0,0 +1,63 @@
+      not_found_in_database: "Invalide %{authentication_keys} of wachtwoord."
+      timeout: "Je sessie is verlopen. Log nogmaals in om door te gaan."
+      unauthenticated: "Je moet inloggen of aanmelden voordat je door kunt gaan."
+      unconfirmed: "Je moet je emailadres bevestigen voordat je door kunt gaan."
+    mailer:
+      confirmation_instructions:
+        subject: "Bevestiging instructies"
+      reset_password_instructions:
+        subject: "Reset wachtwoord instructies"
+      unlock_instructions:
+        subject: "Ontgrendel instructies"
+      password_change:
+        subject: "Wachtwoord veranderd"
+    omniauth_callbacks:
+      failure: "Kan je niet authenticeren via %{kind} omdat \"%{reason}\"."
+      success: "Successvol geauthenticeerd via %{kind} account."
+    passwords:
+      no_token: "Je kan niet op deze pagina komen zonder uit een wachtwoord reset email te komen. Indien je wel uit zo'n email komt, let er op dat je de volledige URL hebt overgenomen."
+      send_instructions: "Binnen enkele minuten zul je een email ontvangen met instructies over hoe je je wachtwoord kunt resetten."
+      send_paranoid_instructions: "ALs je emailadres reeds in onze database staat, dan zul je instructies ontvangen om je wachtwoord te resetten."
+      updated: "Je wachtwoord is ingesteld. Je bent nu ingelogd."
+      updated_not_active: "Je wachtwoord is met succes aangepast."
+    registrations:
+      destroyed: "Tot ziens! Je account is met succes verwijderd. We hepen je in de nabije toekomst weer te zien."
+      signed_up: "Welkom! Je hebt je met succes aangemeld."
+      signed_up_but_inactive: "Je hebt je met succes aangemeld, maar we konden je niet inloggen omdat je account nog niet is geactiveerd."
+      signed_up_but_locked: "Je hebt je met succes aangebeld, maar we konden je niet inloggen omdat je account is vergrendeld."
+      signed_up_but_unconfirmed: "Een bericht met een bevestiging link is via email naar je toegestuurd. Volg alsjeblieft de link om te bevestigen."
+      update_needs_confirmation: "Je hebt je account met succes aangepast, maar we moeten je nieuwe emailadres bevestigen. Controleer je email en volg de bevestiging link om door te gaan."
+      updated: "Je account is met succes bijgewerkt."
+    sessions:
+      signed_in: "Ingelogd."
+      signed_out: "Uitgelogd."
+      already_signed_out: "Uitgelogd."
+    unlocks:
+      send_instructions: "Je zult binnen enkele minuten een email ontvangen met instructies om je account te ontgrendelen."
+      send_paranoid_instructions: "Indien je account bestaat zul je binnen enkele minuten een email ontvangen met instructies om je account te ontgrendelen."
+      unlocked: "Je account is met succes ontgrendelt. Log in om door te gaan."
+  errors:
+    messages:
+      already_confirmed: "was reeds bevestigd, probeer eens in te loggen"
+      confirmation_period_expired: "moet bevestigd worden voor  %{period}, plaats een nieuw verzoek"
```

Comment:
Looks like there is an extra space between "voor" and "%{period}" that needs to be removed.